### PR TITLE
[semver:minor] Allow recursive option on fmt

### PR DESCRIPTION
**Pull request by dmechas**
This will add the possibility of run format recursively based on a parameter

### CODE REVIEW

**Review by gmemstr — APPROVED** [review]

**Comment by dmechas** [comment]
@gmemstr waiting for approval https://app.circleci.com/pipelines/github/CircleCI-Public/terraform-orb/255/workflows/59266e3c-aa2a-4873-a0df-4e9123aec912

**Pull request merged**
